.eslintrc.js

```diff
@@ -0,0 +1,65 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    plugins: [ '@typescript-eslint' ],
+    extends: [
+        'eslint:recommended',
+    ],
+    env: {
+        node: true,
+        es6: true,
+    },
+    globals: {
+        describe: true,
+        xdescribe: true,
+        fdescribe: true,
+        it: true,
+        xit: true,
+        fit: true,
+        expect: true,
```

Comment:
I believe we can remove these when adding `jasmine: true` to `env`

Reply:
Done!